unit-tests/shared/src/test/scala/org/scalanative/testsuite/javalib/lang/MathTest.scala

```diff
@@ -275,4 +275,57 @@ class MathTest {
     }
   }
 
+  @Test def multiplyHighTests(): Unit = {
```

Comment:
CI fails becouse JDK 8 does not define this method. `Math.multipleHight` was added in JDK 9, so we need to put the test in this file https://github.com/scala-native/scala-native/blob/bc2d1cbf04a428b178a35f7cf3f5a7f12a374dc6/unit-tests/shared/src/test/require-jdk9/org/scalanative/testsuite/javalib/lang/MathTestOnJDK9.scala#L6 
The `require-jdkX` ensures that given test is only compiled when using JDK X or later

Reply:
Thanks for the hint! I've moved the test to the linked file. Hopefully all checks are successful.